homeassistant/components/airvisual_pro/config_flow.py

```diff
@@ -60,7 +60,7 @@ async def async_validate_credentials(
     except NodeProError as err:
         LOGGER.error("Unknown Pro error while connecting to %s: %s", ip_address, err)
         errors["base"] = "unknown"
-    except Exception as err:  # pylint: disable=broad-except
+    except Exception as err:  # noqa: BLE001
```

Comment:
Why does this case different than other cases that use `LOGGER.exception`? 
```suggestion
    except Exception as err:
```

Reply:
That's because the LOGGER variable is defined in another file and currently ruff can't do multifile analysis.